data/filters/wg21.py

```diff
@@ -545,6 +549,21 @@ def citation_link(elem, doc):
     elem.url = url
     return elem
 
+def automatic_cross_reference_text(elem, doc):
+    if not isinstance(elem, pf.Link) or ('self-link' in elem.classes) or not (elem.url.startswith('#')):
```

Comment:
I tend to prefer doing `if not (conditions ...)` for filter conditions like this, and the `link_text` condition can go in here as well:
```suggestion
    if not (isinstance(elem, pf.Link) and
            elem.url.startswith('#') and
            ('self-link' not in elem.classes) and
            pf.stringify(elem) == ""):
         return None
```

Reply:
I am not a big fun of it in this very case because we use double negation for `self-link`. But it's a slight preference, so I applied the comment as you suggested